traffic_monitor/static/script.js

```diff
@@ -109,7 +109,14 @@ function getTrafficOpsUri() {
 
 function getTrafficOpsCdn() {
 	ajax("/publish/ConfigDoc", function(r) {
-		document.getElementById("cdn-name").textContent = JSON.parse(r).cdnName || "unknown";
+		let opsConfig = JSON.parse(r);
+		document.getElementById("cdn-name").textContent = opsConfig.cdnName || "unknown";
+		var discIconContainer = document.getElementById("icon-disc-holder");
+		if ((opsConfig.usingDummyTO || true) === false) {
+			discIconContainer.classList.add("hidden");
```

Comment:
This is unreachable, because `(opsConfig.usingDummyTO || true) === false` is tautologically false.

Also, you could directly set the element's [`hidden` attribute](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/hidden) to avoid adding extra CSS rules to express that concept.

Reply:
You are correct, I did not mean to push that up. As for the hidden attribute, I was using that originally but it wasn't working, I'll look into it again.